raiden_installer/utils.py

```diff
@@ -75,17 +75,16 @@ def send_raw_transaction(w3, account, contract_function, *args, **kw):
     return w3.eth.waitForTransactionReceipt(tx_hash, timeout=WEB3_TIMEOUT)
 
 
-def wait_for_transaction(web3: Web3, transaction_receipt: Dict[str, Any]) -> None:
-
+def wait_for_transaction(w3: Web3, transaction_receipt: Dict[str, Any]) -> None:
     if "blockNumber" not in transaction_receipt:
         raise KeyError("blockNumber not in transaction receipt.")
 
     block_with_transaction = transaction_receipt["blockNumber"]
-    current_block = web3.eth.blockNumber
+    current_block = w3.eth.blockNumber
 
     while current_block < block_with_transaction:
```

Comment:
We had the same discussion recently in the services. Can you please add `Confirmation_Interval` to the waiting time. I suggest 5 blocks, should be sufficient. This reduces the chance of error due to reorgs.
(It might make sense to also let the user know because of the increased waiting time)

Reply:
Makes totally sense. Thank you! How would you let the user know? I think we already say in the frontend that it takes time whenever a transaction is processed.

Reply:
Ok then it's fine. Just letting the user know that it might take time since 5 blocks are already > 1 min